_posts/en/newsletters/2020-03-25-newsletter.md

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Bitcoin Optech Newsletter #90'
+permalink: /en/newsletters/2020/03/25/
+name: 2020-03-25-newsletter
+slug: 2020-03-25-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes several questions and answers from the
+Bitcoin StackExchange and describes notable changes to popular Bitcoin
+infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+*No significant news about Bitcoin infrastructure development this week.*
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{%
+endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Eclair #1339][] prevents users from setting their htlc-minimum
+  amount to 0 milli-satoshis. The new minimum is 1 milli-satoshi. This may be
+  influenced by [LND #3697][], which lowered its previous default from 1,000
+  msat to 0 msats to allow sub-satoshi payments as covered in
```

Comment:
I'm confused about this. Both Newsletter 77 and the code comment (https://github.com/lightningnetwork/lnd/pull/3697/files#diff-356ddb2e7efca712327c3b2d94d3afd3R41) say the default is 0, but the PR description (https://github.com/lightningnetwork/lnd/pull/3697) says 1. The code also seems to make 1 the default (https://github.com/lightningnetwork/lnd/pull/3697/files#diff-356ddb2e7efca712327c3b2d94d3afd3R48), but my understanding of LND/Go is not great.

Reply:
Looks like it was changed during the PR from 0 to 1: https://github.com/lightningnetwork/lnd/pull/3697#discussion_r355104498

Reply:
Fixed it! https://github.com/lightningnetwork/lnd/pull/4108